bundles/org.openhab.binding.resol/src/main/resources/OH-INF/i18n/resol_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
Can this file be removed?

Reply:
I was planning to do the translation into German at least and the file is a form of a reminder. But obviously it did not work well...